configuration/expose-portal-over-http.adoc

```diff
@@ -0,0 +1,51 @@
+[id="expose-portal-over-http"]
+= Exposing RHACS portal over HTTP
+include::modules/common-attributes.adoc[]
+:context: expose-portal-over-http
+
+toc::[]
+
+[role="_abstract"]
+Enable an unencrypted HTTP server to expose the RHACS portal through ingress controllers, Layer 7 load balancers, Istio, or other solutions.
+
+If you use an ingress controller, Istio, or a Layer 7 load balancer that prefers unencrypted HTTP back ends, you can configure {product-title} to expose the RHACS portal over HTTP.
+Doing this makes the RHACS portal available over a plaintext back end.
+
+[IMPORTANT]
+====
+To expose the RHACS portal over HTTP, you must be using an ingress controller, a Layer 7 load balancer, or Istio to encrypt external traffic with HTTPS.
+It is insecure to expose the RHACS portal directly to external clients by using plain HTTP.
+====
+//TODO Add a link to the endpoint configuration topic as the recommended way to expose portal.
+
+To specify a HTTP endpoint you must use an `<endpoints_spec>`.
```

Comment:
* a -> an
* I'm not sure if there's a better way to do this, but the first time I read this, I interpreted this to mean that endpoints_spec was a legit literal value (even though it is in the replaeable value format). Looking back at the original stackrox docs, I think it makes more sense there because it's provided right after the command that uses it. I don't know what the right answer is here. This is a lot of information to include in a callout or something after the command(s); plus you'd technically need to do it twice then, one for each module. I'd try to play around with it a little more. If you do keep it here, maybe make it part of a prereqs area of the assembly and/or refer to it more generally at first. Like that you'll have to specify the endpoints, noted as `<endpoints_spec>` in following procedures. Actually I don't know if I like that either :). Usually I try to clean up my comments after I've thought about them more, but I don't have anything more helpful to suggest on how to handle this. After all that, I think that the right answer is to put it after each of the commands in the procedures, if you're able to make it more concise. I'll stop typing now and will let you take a look :)

Reply:
Nah! Thanks for keeping this whole comment, it really made me rethink what I'm doing here. Till now I was keeping myself from adding xrefs in modules but I did it for this one.

I don't want to repeat this portion for each command because the page is not that big. I like your suggestion of having the explanation in Prerequisites section. Looks good to me now.

Reply:
@gaurav-nelson This does look better to me now. Though, I can't merge with an xref in a module, per our guidelines. So unless you want to ask Vikram for an exception ;) you'd have to take that out and add your // TODO. Sorry :/

Reply:
I confirmed with Vikram, we can't use `xref` yet. I'll remove those.